### DISCUSSION TITLE
Setting to always run in background

### DISCUSSION THREAD

**Discussion by zibra** [NightscoutFoundation/xDrip#2775] [Q&A]
Is it possible to setup app somehow so it will always run in background? I mean to restart automatically after crash or closing by mistake. In such cases I'm loosing data transfer from Juggluco and AAPS which xDrip is proxying

**Comment by Navid200** [comment]
I don't understand the logic of the question.  If the app crashes, it means that it has stopped.  If it restarts automatically, it will not stop it from crashing.

Anyway, the only thing I know that may be related is that you can set xDrip to start automatically after you start the phone.  
The setting is:

Settings &#8722;> xDrip+ Display Settings &#8722;> Show xDrip on Boot.
I have noticed that it actually starts xDrip automatically even if I force stop it.  But, I have not done enough testing to confirm this part.

**Comment by jamorham** [comment]
xDrip already does a lot of things to ensure it is always running. Problems in this area I think would be likely phone specific. Which phone model are you using?
